src/components/application_manager/src/rpc_service_impl.cc

```diff
@@ -451,6 +532,8 @@ void RPCServiceImpl::SendMessageToMobile(
     LOG4CXX_WARN(logger_, "Can't send msg to Mobile: failed to create string");
     return;
   }
+  const auto api_function_id = static_cast<mobile_apis::FunctionID::eType>(
+      (*message)[strings::params][strings::function_id].asInt());
```

Comment:
Should this be asUInt()? Referencing the removed line below.

Reply:
fixed in 6a23e0c4b08b374cefd63c236415371a0b0e2a32